nservicebus/outbox/index.md

```diff
@@ -79,8 +79,64 @@ In order to gradually convert an entire system from the DTC to the Outbox:
 WARNING: When verifying outbox functionality, it can sometimes be helpful to temporarily [stop the MSDTC Service](https://technet.microsoft.com/en-us/library/cc770732.aspx). This ensures that the Outbox is working as expected, and no other resources are enlisting in distributed transactions.
 
 
+## Message identity and idempotent processing
+
+If handlers are idempotent then outbox might not be required at all. Outbox relies on message identity to guarantee consistency. When using outbox on both the sender and receiver message identifiers are consistent if a handler is invoked more-then-once for a message. 
```

Comment:
> When using outbox on both the sender and receiver message identifiers are consistent if a handler is invoked more-then-once for a message. 

I don't understand this sentence or what it's trying to tell me. Also, more-than-once.